xdsl/frontend/pyast/type_conversion.py

```diff
@@ -12,11 +13,18 @@
     _FrontendType,  # pyright: ignore[reportPrivateUsage]
 )
 from xdsl.frontend.pyast.exception import CodeGenerationException
-from xdsl.ir import Attribute
+from xdsl.ir import Attribute, TypeAttribute
 
 TypeName: TypeAlias = str
 
 
+class SourceIrTypePair(NamedTuple):
```

Comment:
```suggestion
class SourceIRTypePair(NamedTuple):
```

I wasn't pinged, so just caught my eye while scrolling through in a dash.
We have named things in core as `IRNode` etc., hence my nit above.